httpie/internal/daemons.py

```diff
@@ -87,10 +87,7 @@ def _spawn_posix(args: List[str], process_context: ProcessContext) -> None:
     if platform.system() == 'Darwin':
         # Double-fork is not reliable on MacOS, so we'll use a subprocess
         # to ensure the task is isolated properly.
-        process = _start_process(args, env=process_context)
-        # Unlike windows, since we already completed the fork procedure
-        # we can simply join the process and wait for it.
-        process.communicate()
```

Comment:
Waited for the process and could send some data to it or return its return code - but nothing of the sort is being done, basically blocking until the process finishes. But since the purpose of the process is to be daemonized (or possibly daemonized already), it doesn't seem to make sense.